src/tools/handle-url-query.js

```diff
@@ -11,21 +18,31 @@ async function loadJsmdFromNotebookUrl(url) {
     const parser = new DOMParser()
     const doc = parser.parseFromString(notebookString, 'text/html');
     const jsmd = doc.querySelector('#jsmd').innerHTML
-    store.dispatch(importFromURL(stateFromJsmd(jsmd))).then(() => {
-      if (store.getState().viewMode === 'presentation') { store.dispatch(evaluateAllCells(store.getState().cells, store)) }
-    })
+    loadJsmd(jsmd)
   } catch (err) {
     console.error('failed to load notebook url', err);
   }
 }
 
+async function loadIpynbFromUrl(url) {
+  try {
+    const response = await fetch(url)
+    const ipynbJson = await response.json()
+    const jsmd = importIpynb(url, ipynbJson)
+    loadJsmd(jsmd)
+  } catch (err) {
+    console.error('failed to load ipynb url', err)
```

Comment:
Any chance we could use `updateAppMessage` from `actions.jsx` here once the snackbar PR #617 lands? I'm assuming that PR might land before this one. Either that or this can be tackled in another issue easily.

Reply:
Sure, I don't see why that wouldn't work.